dpbench/config/__init__.py

```diff
@@ -39,6 +51,8 @@ def read_configs(dirname: str = os.path.dirname(__file__)) -> Config:
         benchmark = Benchmark.from_dict(bench_info.get("benchmark"))
         C.benchmarks.append(benchmark)
 
+
+def read_frameworks(C: Config, framework_info_dir: str):
```

Comment:
`C` is an unclear name. How about just `config: Config`? Similar comment for other places.